Update installed software.

hwartig opened the pull request:
Most notably I updated ruby to 2.5.

As we anyway need to increase the major version for this I'm considering to also update the
baseimage to a more recent ubuntu version and thereby fixing some of the vulnerability alerts shown
in https://hub.docker.com/r/ad2games/docker-rails/tags/latest/

hwartig committed:
Update installed software.

tsujigiri reviewed (approved):
💎

guigs commented on base/Dockerfile:
```diff
@@ -36,7 +36,8 @@ RUN curl -o ruby.tgz \
 RUN echo 'gem: --no-document' > $HOME/.gemrc
 RUN gem update --system ${RUBYGEMS_VERSION}
-RUN gem install -v ${BUNDLER_VERSION} bundler
+# we have to force install bundler as it is a vendored gem since rubygems 2.7.0
```
Can't we just rely on the vendored bundler then?

hwartig replied:
I'm a fan of using latest stable. Do you see any downsides?

guigs replied:
No, I'm fine keeping it.

guigs commented on README.md:
```diff
@@ -30,10 +30,13 @@ To use it in your app, create a `Dockerfile` with the following content:
+might be useful if you want to test against the bleeding edge in your testing/beta environment.
+
+From version 3.0 docker-rails uses ruby 2.5. If you still use ruby 2.4 you have to use docker-rails
+versions 2.x. We also provide `ruby-2.5` and `ruby-2.3` tags that will point to the latest released
```
I believe you meant `ruby-2.4` instead of `ruby-2.3`

guigs replied:
Anyway, nice idea of providing `ruby-2.x` tags :+1:

hwartig replied:
nice catch thanks

tsujigiri committed:
Merge branch 'master' into ruby-2.5

guigs commented on CHANGELOG.md:
```diff
@@ -4,6 +4,12 @@
 
 - Add `file` command
 
+- Upgrade Ruby (2.5.0)
```
Since this will be for next release, I think these entries should be added before `## 2.7.3` under something like `## unreleased`.

Then, when running `release.sh` it will open this for edit, then we change `## unreleased` to `## 2.8.0`

guigs commented on .ruby-version:
```diff
@@ -1 +1 @@
-2.4.3
```
Let's move to 2.5.1

guigs commented:
Hey, let's upgrade once more, merge and release this?

tsujigiri committed:
Update baseimage to 0.11 (#24)

This updates OpenSSL from 1.0.1 to 1.1.0 and resolves the SSL issues we have with CAKE's API*.

* https://app.bugsnag.com/ad2games/backoffice/errors/5bb96b06a48b3a001aeceff7
